types/node/ts4.0/node-tests.ts

```diff
@@ -0,0 +1 @@
+import '../3.6/node-tests';
```

Comment:
Originally I tried `import '../node-tests' but then `tsc` failed due to "expected errors" (`$ExpectError`). It looks like the other version folders don't bother running tests which contain expected errors, so I just copied what they were doing.